.github/workflows/build_debian.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
             R2_INCOMING_BUCKET: ${{ vars.R2_INCOMING_BUCKET }}
```

Comment:
should we keep this file?

Reply:
No...

I want to move tchap jobs into a separate directory and then just leave all the element files. That way we spend zero time merging files that are deleted or not deleted. https://github.com/tchapgouv/tchap-web-v4/issues/761
Not done yet, I want to keep it separate from a merge PR.

So I'm going to leave them for now.